src/components/onboarding/hooks/useOnBoardingStepsOrder.ts

```diff
@@ -32,6 +32,8 @@ export default function useOnBoardingStepsOrder(
       : transactionsCount < 20
     if (showEnergyStep) usedSteps.push('energy')
 
+    if (usedSteps[usedSteps.length - 1] === 'energy') usedSteps.push('signer')
```

Comment:
why do you need to check if prev step is energy?

Reply:
Oh, I understand what you meant. Will refactor to check if account has already sign or not.

Reply:
@teodorus-nathaniel Please check 6227aa1